TTBot/Program.cs

```diff
@@ -149,6 +149,20 @@ async Task CancelSignup(string reason)
                 return;
             }
 
+
+            if (reaction.User.Value is IGuildUser user) {
+                var moderatorService = _serviceProvider.GetRequiredService<IModerator>();
+                if ((user.GuildPermissions.ManageGuild || await moderatorService.GetLeaderboardModeratorAsync(user.GuildId, user.Id) != null) 
```

Comment:
` await moderatorService.GetLeaderboardModeratorAsync(user.GuildId, user.Id)`

I dont think this is right. The 2nd parameter is meant to be a role id. I think you want to use `IModerator.GetLoaderboardModeratorsAsync` which will return the moderator roles then check if the user has one of those roles.